class Solution {
    public int countIntersectingIntervals(int[][] intervals) {
        int cnt=0;
        int n=intervals.length;
        for(int i=0;i<n-1;i++){
            for(int j=i+1;j<n;j++){
                if(Math.max(intervals[i][0],intervals[j][0])<=Math.min(intervals[i][1],intervals[j][1])) cnt++;
            }
        }
        return cnt;
    }
}